docs/logging.md

```diff
@@ -72,4 +72,8 @@ Also all unhandled exceptions are redirected to Cloud Logging too:
 bigflow.log.init_workflow_logging(workflow)
 raise ValueError("message")
 ```
-The code above contains an unhandled `ValueError` exception, which will be also available at Cloud Logging.
\ No newline at end of file
+The code above contains an unhandled `ValueError` exception, which will be also available at Cloud Logging.
+
+
+## Bigflow logs command
+It is possible to create link to CL through CLI command [`logs`](./cli.md#generating-link-to-gcp-logging).
```

Comment:
To access logs, use the CLI command [`bigflow logs`](./cli.md#generating-link-to-gcp-logging).